front/pages/api/v1/w/[wId]/data_sources/[name]/tables/[tId]/parents.ts

```diff
@@ -0,0 +1,156 @@
+import type { WithAPIErrorResponse } from "@dust-tt/types";
+import { CoreAPI } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import config from "@app/lib/api/config";
+import { getDataSource } from "@app/lib/api/data_sources";
+import { Authenticator, getAPIKey } from "@app/lib/auth";
+import logger from "@app/logger/logger";
+import { apiError, withLogging } from "@app/logger/withlogging";
+
+export type PostParentsResponseBody = {
+  updated: true;
+};
+
+/**
```

Comment:
This endpoint should not be documented as we don't want to open the table related endpoints yet.

Reply:
eslint says all endpoints need to be documented ? (and all here indeed)

Reply:
Except for the one around tables 🙃.